python/nano/src/bigdl/nano/utils/pytorch/metadata.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+from ..common import invalidInputError
+
+
+class BaiscMetaDataInfo(object):
+    """Meta info class for basic type like int, float, str
+    """
+    def __init__(self, input_element):
+        self.type = type(input_element)
+        self.reconstruct = False
+
+
+class TensorMetaDataInfo(object):
+    """Meta info class for torch.Tensor
+    """
+    def __init__(self, input_element):
+        self.type = type(input_element)
+        self.shape = input_element.shape
+        self.reconstruct = False
```

Comment:
so this might be renamed to `self.need_reconstruct` to make this easier to understand for later developer.

Reply:
done